simple/sample/output/observations_countries.csv

```diff
@@ -1,14 +1,14 @@
-dcid,date,var1,var2
-country/AFG,2023,0.19,6
-country/YEM,2023,0.21,56
-country/AGO,2023,0.29,6
-country/ZMB,2023,0.31,34
-country/ZWE,2023,0.37,76
-country/ALB,2023,0.5,34
-wikidataId/Q22062741,2023,0.5,97
-country/DZA,2023,0.52,92
-country/AND,2023,0.76,9
-country/ASM,2023,,34
-country/AIA,2023,,42
-country/WLF,2023,,75
-country/ESH,2023,,65
+dcid,date,var1,var2,provenance
+country/AFG,2023,0.19,6,custom/base/Prov_1
```

Comment:
custom and base mean opposite things?

either:  `dc/custom/Prov_1` or `custom/p/Prov_1` 

Prefer the latter for symmetry with custom/s/src

Reply:
Done.